backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs

```diff
@@ -20,6 +25,361 @@ public static void Configure(this ModelBuilder modelBuilder)
+                    Country = "Poland",
+                    CreatedAt = new DateTimeOffset(new DateTime(2021, 7, 26)),
+                    Description = "It has survived not only five centuries, but also the leap into electronic, remaining essentially unchanged.",
+                    Email = "moderator@gmail.com",
+                    Gender = false,
+                    GroupId = null,
+                    IconURL = avatar,
+                    Salt = "jdEeMqVH6VdFQjdzLTWNT4+vHS2B3MWNQQTUYm9o984=",
+                    Password = HashPassword(password, Convert.FromBase64String("jdEeMqVH6VdFQjdzLTWNT4+vHS2B3MWNQQTUYm9o984=")),
+                    UserName = "moderator"
+                }
+            };
+        }
+
+        public static IList<Album> GenerateAlbums(IList<User> users) =>
+            new List<Album>
+            {
+                new Album
+                {
+                    Id=1,
+                    AuthorId=users[1].Id,
+                    AuthorType=AuthorType.Artist,
+                    CreatedAt= new DateTimeOffset(new DateTime(2020, 9, 9)),
+                    Description="Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+                    GroupId=null,
+                    IconURL= "https://upload.wikimedia.org/wikipedia/ru/6/69/ImagineCover.jpg",
+                    isPublished=true,
+                    isSingle=true,
+                    Name="Imagine",
+                    Region= AlbumRegion.UK,
+                    ReleaseYear = 1971
+                },
+                new Album
+                {
+                    Id=2,
+                    AuthorId=users[1].Id,
+                    AuthorType=AuthorType.Artist,
+                    CreatedAt= new DateTimeOffset(new DateTime(2020, 9, 9)),
+                    Description="Lorem printing and typesetting industry.",
+                    GroupId=null,
+                    IconURL= "https://upload.wikimedia.org/wikipedia/ru/2/2a/MindGames.jpg",
+                    isPublished=true,
+                    isSingle=false,
+                    Name="Mind Games",
+                    Region= AlbumRegion.UK,
+                    ReleaseYear = 1973
+                },
+                new Album
+                {
+                    Id=3,
+                    AuthorId=users[2].Id,
+                    AuthorType=AuthorType.Artist,
+                    CreatedAt= new DateTimeOffset(new DateTime(2020, 9, 9)),
+                    Description="Lorem text of the printing and typesetting industry.",
+                    GroupId=null,
+                    IconURL= "https://upload.wikimedia.org/wikipedia/ru/thumb/e/eb/Mccartney_album.jpg/274px-Mccartney_album.jpg",
+                    isPublished=true,
+                    isSingle=true,
+                    Name="McCartney",
+                    Region= AlbumRegion.UK,
+                    ReleaseYear = 1970
+                },
+            };

```

Comment:
![image](https://user-images.githubusercontent.com/24195200/127611775-a429cbf7-0bc4-4620-ad5a-6f23da8563b5.png)
Take care of spaces, this is a small issue, but still, need your attention. The third variant is the best. If you use Visual Studio you could use this is a combination for formatting: CTRL + K + F

Reply:
formatting fixed